CHANGELOG-0.10.md

```diff
@@ -7,6 +7,7 @@
 - [#1798](https://github.com/epiphany-platform/epiphany/issues/1798) - Additional alerts for Prometheus
 - [#1355](https://github.com/epiphany-platform/epiphany/issues/1355) - Updating cloud based OS images - configuration required for Azure RHEL LVM images
 - [#2081](https://github.com/epiphany-platform/epiphany/issues/2081) - Replace Skopeo with Crane
+- [#2099](https://github.com/epiphany-platform/epiphany/issues/2099) - Remove the Crane binary from the repo
```

Comment:
Not so important, but in latest check list we have such point: `Changelog updated (if affected version was released)`

Reply:
Not sure if I get that, so we should add changelog only when we release it?

Reply:
IMO (@to-bar thinks the same) there is no need to add second changelog entry since this feature is introduced in the same (not already released) version 0.10.0.

Reply:
Removed.